File: Backend/BankingPaymentsApp_API/BankingPaymentsApp_API/Models/ProofType.cs
using System.ComponentModel.DataAnnotations;

namespace BankingPaymentsApp_API.Models
{
    public enum DocProofType
    {
        IDENTITY_PROOF,
        ADDRESS_PROOF,
        DATE_OF_BIRTH_PROOF,
        PHOTOGRAPH,
        PAN_CARD
    }
    public class ProofType
    {
        [Key]
        public int ProofId { get; set; }
        [Required(ErrorMessage = "Type is Required!")]
        public AccStatus Type { get; set; }
    }
}

File: frontend/BankingPaymentsApp_Frontend/src/Models/ProofType.ts
export enum DocProofType {
  IDENTITY_PROOF = "IDENTITY_PROOF",
  ADDRESS_PROOF = "ADDRESS_PROOF",
  DATE_OF_BIRTH_PROOF = "DATE_OF_BIRTH_PROOF",
  PHOTOGRAPH = "PHOTOGRAPH",
  PAN_CARD = "PAN_CARD",
  OTHER = "OTHER",
}

export interface ProofType {
  TypeId: number;
  Type: DocProofType;
}